doc/source/reference/c-api/array.rst

```diff
@@ -3435,6 +3450,14 @@ Other constants
     The return value of successful converter functions which are called
     using the "O&" syntax in :c:func:`PyArg_ParseTuple`-like functions.
 
+.. c:macro:: NPY_RAVEL_AXIS
+
+    Some NumPy functions (mainly the C-entrypoints for Python functions)
+    have an ``axis`` argument.  This macro may be passed for ``axis=None``.
+
+    .. note::
+        This macro is a NumPy version dependent at runtime.
```

Comment:
```suggestion
        This macro is NumPy version dependent at runtime.
```

Maybe also a sentence explaining why it depends on Numpy version and/or why that matters?